os-datavirt/added/standalone-openshift.xml

```diff
@@ -525,14 +525,14 @@
             <preparedplan-cache infinispan-container="teiid-cache"/>
             <transport name="embedded"/>
             <transport name="odata">
-                <authentication security-domain="teiid-security"/>
+                <authentication security-domain="##ODATA_SECURITY_DOMAIN##"/>
```

Comment:
I don't think so - Teiid exposes JDBC, ODBC, REST/ODATA, and "embedded". We are covering and exposing all but the last.